--- urlview-0.9/url_handler.sh.deush	Tue Oct 24 14:21:07 2000
+++ urlview-0.9/url_handler.sh	Tue Oct 24 14:21:46 2000
@@ -28,9 +28,9 @@
 # VT: Launch in the same terminal
 
 # The lists of programs to be executed are
-https_prgs="/usr/X11R6/bin/netscape:XW /usr/bin/lynx:XT"
-http_prgs="/usr/bin/lynx:XT /usr/X11R6/bin/netscape:XW"
-mailto_prgs="/usr/bin/mutt:VT /usr/bin/elm:VT /usr/bin/pine:VT /usr/bin/mail:VT"
+https_prgs="/usr/bin/firefox:XW /usr/bin/lynx:XT"
+http_prgs="/usr/bin/firefox:XW /usr/bin/lynx:XT"
+mailto_prgs="/usr/bin/mutt:VT /usr/bin/elm:VT /usr/bin/pine:VT /bin/mail:VT"
 gopher_prgs="/usr/bin/lynx:XT /usr/bin/gopher:XT"
 ftp_prgs="/usr/bin/lynx:XT /usr/bin/ncftp:XT"
 
